### PR TITLE
Update support for new architecture ARM devices and added support for more OS.

### PR DESCRIPTION
**Pull request by quydang04**
### **Pull Request Summary:**

In this pull request, I've made significant enhancements to broaden the compatibility and usability of 3x-ui:

1. **Expanded Device Support:**
   - Added support for Armbian and Manjaro, enabling 3x-ui to run on a wider range of devices.
   - Specifically addressed compatibility for devices such as the Orange Pi Zero LTS and Android TV Box, commonly used in Vietnam.

2. **New Architecture Support:**
   - Introduced support for ARM32 (armv7) architecture, catering to older devices like the Orange Pi Zero LTS, which utilizes Armbian and Allwinner H3 (armv7).
   - This ensures that 3x-ui can now seamlessly run on these previously incompatible devices.

3. **Enhanced Installation Process:**
   - Implemented a pre-installation command that prompts users to check for the latest packages and updates across all supported operating systems.
   - This proactive step ensures a smooth installation process by aligning 3x-UI with the latest dependencies, enhancing overall stability and performance.

I kindly request your thorough review and consideration of these changes for incorporation into the next version of 3x-ui. These additions aim to make 3x-ui more accessible and functional for a diverse user base. Your valuable insights and feedback are greatly appreciated.

Thank you very much.

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -10,8 +10,8 @@ jobs:
   build:
     strategy:
       matrix:
-        platform: [amd64, arm64]
-    runs-on: ubuntu-20.04
+        platform: [amd64, arm64, arm]
+    runs-on: ubuntu-latest
```
Comment (MHSanaei) [inline .github/workflows/release.yml:14]: don't change runs-on please 
undo your changes

File: install.sh [hunk 2]
```diff
@@ -146,7 +151,7 @@ install_x-ui() {
     cd x-ui
     chmod +x x-ui bin/xray-linux-$(arch3xui)
     cp -f x-ui.service /etc/systemd/system/
-    wget --no-check-certificate -O /usr/bin/x-ui https://raw.githubusercontent.com/MHSanaei/3x-ui/main/x-ui.sh
+    wget --no-check-certificate -O /usr/bin/x-ui https://raw.githubusercontent.com/MHSanaei/x-ui/main/x-ui.sh
```
Comment (MHSanaei) [inline install.sh]:
```suggestion
    wget --no-check-certificate -O /usr/bin/x-ui https://raw.githubusercontent.com/MHSanaei/3x-ui/main/x-ui.sh
```

File: .github/workflows/release.yml (hunk 1 above)
Comment (MHSanaei) [inline .github/workflows/release.yml]:
```suggestion
    runs-on: ubuntu-20.04
```

**Comment by MHSanaei** [comment]
thanks for your PR i'll merge it for next update

**Comment by quydang04** [comment]
Okay, thank you very much

**Pull request merged**